### PR TITLE
Updates from package template

### PR DESCRIPTION
**Pull request by Cadair**
This PR has been generated by a script, it should update the repo with the latest changes from the package template.

### CODE REVIEW

File: .cruft.json [hunk 1]
```diff
@@ -10,13 +10,20 @@
...
+      "issue_tracker_url": "https://github.com/DKISTDC/dkist/issues",
       "license": "BSD 3-Clause",
       "minimum_python_version": "3.10",
       "use_compiled_extensions": "n",
       "enable_dynamic_dev_versions": "y",
       "include_example_code": "n",
       "include_cruft_update_github_workflow": "y",
-      "_sphinx_theme": "alabaster",
+      "use_extended_ruff_linting": "y",
+      "_sphinx_theme": "sunpy",
```
Comment (Cadair) [inline .cruft.json:26]:
```suggestion
      "_sphinx_theme": "dkist",
```

File: dkist/_dev/scm_version.py [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # Try to use setuptools_scm to get the current version; this is only used
 # in development installations from the git repository.
-import os.path
+from pathlib import Path
 
 try:
     from setuptools_scm import get_version
 
-    version = get_version(root=os.path.join("..", ".."), relative_to=__file__)
+    version = get_version(root=Path('../..'), relative_to=__file__)
```
Comment (SolarDrew) [inline dkist/_dev/scm_version.py:8]: Won't this break on Windows?

File: pyproject.toml [hunk 3]
```diff
@@ -77,7 +78,12 @@ docs = [
 ]
 
 [project.urls]
-repository = "https://github.com/DKISTDC/dkist"
+Homepage = "https://github.com/DKISTDC/dkist"
+"Source Code" = "https://github.com/DKISTDC/dkist"
+Download = "https://pypi.org/project/dkist"
+Documentation = "https://docs.dkist.nso.edu/projects/python-tools/en/stable/"
+Changelog = "https://docs.dkist.nso.edu/projects/python-tools/en/stable/whatsnew/changelog.html"
+"Issue Tracker" = "https://github.com/DKISTDC/dkist/issues"
```
Comment (SolarDrew) [inline pyproject.toml:86]: This info is all duplicated in .cruft.json, is that deliberate?

**Review by SolarDrew — COMMENTED** [review]
A couple of minor comments, otherwise looks fine.

**Comment by SolarDrew** [comment]
> Merging https://github.com/DKISTDC/dkist/pull/472 will degrade performances by 18.29%

_This_ concerns me a bit though...

File: dkist/_dev/scm_version.py (hunk 2 above)
Comment (Cadair) [reply dkist/_dev/scm_version.py:8]: nah, Path will translate it on windows.

File: pyproject.toml (hunk 3 above)
Comment (Cadair) [reply pyproject.toml:86]: `.cruft.json` is the source and this is where they get used.

**Comment by Cadair** [comment]
> > Merging #472 will degrade performances by 18.29%
> 
> _This_ concerns me a bit though...

yeah I have no idea why :thinking:

**Comment by Cadair** [comment]
> > > Merging #472 will degrade performances by 18.29%
> > 
> > 
> > _This_ concerns me a bit though...
> 
> yeah I have no idea why 🤔

From a cursory glance at the dashboard it looks like a dask regression.

**Comment by SolarDrew** [comment]
> From a cursory glance at the dashboard it looks like a dask regression.

Then I suggest we pin dask so it doesn't make all our stuff really slow?

**Comment by Cadair** [comment]
Turns out it's an astropy regression caused by https://github.com/astropy/astropy/pull/15474

**Pull request merged**